packages/core/src/components/card/card.tsx

```diff
@@ -83,65 +67,56 @@ export class Card implements ComponentInterface {
       }
     }
 
-    return this.dsoCardClicked.emit({ originalEvent: e, isModifiedEvent: isModifiedEvent(e) });
+    return this.dsoCardClick.emit({ originalEvent: e, isModifiedEvent: isModifiedEvent(e) });
   }
 
   get selectableSlottedElement() {
     return this.host.querySelector("[slot='selectable']");
   }
 
-  get imageSlottedElement() {
-    return this.host.querySelector("[slot='image']");
-  }
-
-  get headingSlottedElement() {
-    return this.host.querySelector("[slot='heading']");
-  }
-
   get interactionsSlottedElement() {
     return this.host.querySelector("[slot='interactions']");
   }
 
   render() {
     const isSelectable = this.selectableSlottedElement !== null;
-    const hasImage = this.imageSlottedElement !== null;
 
     return (
-      <Host onClick={(e: MouseEvent) => this.clickEventHandler(e)} is-selectable={isSelectable} has-image={hasImage}>
-        <div class="dso-card-selectable" hidden={!isSelectable}>
-          <slot name="selectable" />
-        </div>
-        <div class="dso-card-image" hidden={!hasImage}>
-          <slot name="image" />
-        </div>
-        <div class="dso-card-heading">
-          {this.headingSlottedElement instanceof HTMLAnchorElement || !this.href ? (
-            <slot name="heading" />
-          ) : (
-            ((!this.mode || !["download", "extern"].includes(this.mode)) && (
-              <a href={this.href} class="heading-anchor">
-                <slot name="heading" />
-                <dso-icon icon="chevron-right"></dso-icon>
-              </a>
-            )) ||
-            (this.mode === "extern" && (
-              <a href={this.href} class="heading-anchor" target="_blank" rel="noopener noreferrer">
+      <Host is-selectable={isSelectable}>
```

Comment:
Kan iemand (Eric/Hans) onderzoeken of de styling die op dit attribuut wordt geplaatst nog nodig is?

Reply:
Ja, want de selectable wordt nog niet goed in de Grid getoond:

<img width="1503" alt="Screenshot 2024-10-07 at 11 35 22" src="https://github.com/user-attachments/assets/33f7b996-031b-42ca-92ef-77eb83a07979">


Ik denk een @hansgrimm je ;-)

Reply:
en dat komt doordat de hele `display: grid` is losgelaten op `is-selectable`. why?

Reply:
Volgens mij hebben we dit inmiddels opgelost